### PR TITLE
Update deprecated escape sequence

### PR DESCRIPTION
**Pull request by FelixHernandez**
## Update deprecated escape sequence
[sc-2671]

**Description**

This change updates the code, removing the deprecated escape sequence before this produces a bug.
Also the filters now only send the parameters requested as the following image shows.

**Test results**
image

### CODE REVIEW

File: dojo/templates/dojo/filter_snippet.html [hunk 1]
```diff
@@ -54,3 +54,32 @@
     </form>
 
 </div>
+<script>
```
Comment (cneill) [inline dojo/templates/dojo/filter_snippet.html:57]: This snippet appears to be unrelated to the escape sequence fixes?

File: dojo/user/validators.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def get_help_text(self):
 
 class SymbolValidator(object):
     def validate(self, password, user=None):
-        contains_special_character = re.findall('[()[\]{}|\\`~!@#$%^&*_\-+=;:\'\",<>./?]', password)  # noqa W605
+        contains_special_character = re.findall(r'[(){}\[\]|~!@#$%^&*_\-+=;:\'",<>\./?]', password)
```
Comment (cneill) [inline dojo/user/validators.py:78]: Looks like the backtick (``` ` ```) is missing here

**Review by cneill — CHANGES_REQUESTED** [review]

File: dojo/user/validators.py (hunk 2 above)
Comment (FelixHernandez) [reply dojo/user/validators.py]: I added the backtick in the [commit](https://github.com/DefectDojo/django-DefectDojo/pull/8879/commits/1e51a6351795768c064e63a76d213517375b983f)

**Review by mtesauro — APPROVED** [review]
Approved

**Review by cneill — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

**Review by Maffooch — APPROVED** [review]
